src/styles/components/_tool-tip.scss

```diff
@@ -0,0 +1,19 @@
+@import 'variables.scss';
+
+.tool-tip {
```

Comment:
why you have one class separated with "-" and other with "_" ? plz use one style-guide

Reply:
if class name consists of too words instead of using camel case I use "-"
And "_" is used to add child classes. 

Example: 
<div class="Recipe-card">
   <p class="Recipe-card_title>Title</p>
</div>

Reply:
but in this case tooltip must be written together. I'll fix it